README.md

```diff
@@ -79,27 +79,21 @@ To use this script you must have AWS CLI and jq
 	$ chmod +x make_cluster.sh state_cluster.sh
 	```
 
-4. Choose the type of machine to use in the cluster
-
-   Now you can choose what kind of machine: you must take note of the code name.
-   See the [catalog](https://aws.amazon.com/ec2/instance-types/)
-
-
```

Comment:
This whole point is actually a requirement for _Create a cluster_ paragraph: that's the reason the catalog link has been moved to line 88.